src/Resources/contao/templates/dcfe_general_edit.html5

```diff
@@ -49,3 +50,14 @@
         </div>
     </div>
 </form>
+<?php
```

Comment:
I don't like the solution. The class `notsave` is part of MetaModels and not used by the DC General anywhere. Therefore I think the JS schould be part of MM frontend editing also.